packages/app/src/app/pages/Sandbox/Editor/Workspace/index.tsx

```diff
@@ -15,7 +17,8 @@ import {
   ItemTitle,
   VersionContainer,
 } from './elements';
-import ConfigurationFiles from './items/ConfigurationFiles';
+import ConfigurationFiles from './items/ConfigurationFilesOld';
+import { ConfigurationFiles as ConfigurationFilesNew } from './items/ConfigurationFiles';
```

Comment:
I was wondering if we should create 2 folders for this?

/items and /pages and then delete items when done?